services/app-web/src/pages/GraphQLExplorer/GraphQLExplorer.tsx

```diff
@@ -0,0 +1,144 @@
+        __schema {
+            queryType {
+                name
+            }
+            mutationType {
+                name
+            }
+            subscriptionType {
+                name
+            }
+            types {
+                ...FullType
+            }
+            directives {
+                name
+                description
+
+                locations
+                args {
+                    ...InputValue
+                }
+            }
+        }
+    }
+
+    fragment FullType on __Type {
+        kind
+        name
+        description
+
+        fields(includeDeprecated: true) {
+            name
+            description
+            args {
+                ...InputValue
+            }
+            type {
+                ...TypeRef
+            }
+            isDeprecated
+            deprecationReason
+        }
+        inputFields {
+            ...InputValue
+        }
+        interfaces {
+            ...TypeRef
+        }
+        enumValues(includeDeprecated: true) {
+            name
+            description
+            isDeprecated
+            deprecationReason
+        }
+        possibleTypes {
+            ...TypeRef
+        }
+    }
+
+    fragment InputValue on __InputValue {
+        name
+        description
+        type {
+            ...TypeRef
+        }
+        defaultValue
+    }
+
+    fragment TypeRef on __Type {
+        kind
+        name
+        ofType {
+            kind
+            name
+            ofType {
+                kind
+                name
+                ofType {
+                    kind
+                    name
+                    ofType {
+                        kind
+                        name
+                        ofType {
+                            kind
+                            name
+                            ofType {
+                                kind
+                                name
+                                ofType {
+                                    kind
+                                    name
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+`
+
+export const GraphQLExplorer = () => {
+    const endpointUrl = 'http://localhost:3030/local/graphql'
+    const { loggedInUser } = useAuth()
+    const { loading, error, data } = useQuery(INTROSPECTION_QUERY)
+
+    if (loading || !data) {
+        return <Loading />
+    }
+
+    if (error || !loggedInUser) {
+        return <GenericErrorPage />
+    }
+
+    return (
+        <div className={styles.background}>
+            <ApolloExplorer
+                className={styles.explorer}
+                endpointUrl={endpointUrl}
+                schema={data}
+                initialState={{
+                    headers: {
+                        'cognito-authentication-provider':
+                            JSON.stringify(loggedInUser),
```

Comment:
@mojotalantikite This sets up authorization for making the query and works on `local` and `dev`. My question is, is this going to work on `val`?

Reply:
I hope this isn’t doing anything in the review apps, this should only work in local dev. But if this PR is working in the review app, I kind of expect it to work in dev/val/prod b/c it means that its sending the actual auth cookie with the request.

Reply:
`I hope this isn’t doing anything in the review apps`

I thought this was supposed to work in all environments except `prod`? That way acceptance testing could use this.

Reply:
I'm not able to make a request work in the review app, first hurdle is that CORS is blocking that 'cognito-authentication-provider' header, so I think you should only add that in local dev.